src/ncp-spinel/SpinelNCPControlInterface.cpp

```diff
@@ -183,7 +183,7 @@ SpinelNCPControlInterface::config_gateway(bool defaultRoute, const uint8_t prefi
     const static int kDefaultRouteFlag = 1 << 1;
     const static int kDhcpFlag = 1 << 3; (void)kDhcpFlag;
 
-	spinel_ipv6addr_t addr = {};
+	struct in6_addr addr = {};
 	uint8_t flags = 0;
```

Comment:
This is ok/correct. Just curios why the type changes for addr? 
If I am not mistaken `spinel_ipv6adr_t` is `typedef`ed as `struct in6_addr` in spinel.h